README.md

```diff
@@ -43,7 +43,7 @@ the distributed effort. But you can still play, especially if you are patient.
 
 ### Windows
 
-Head to the Github releases page at https://github.com/leela-zero/leela-zero/releases,
+Head to the [Github releases page](https://github.com/gcp/leela-zero/releases),
```

Comment:
URL change?

Reply:
yes indeed, will commit

Reply:
will do an all in one readme commit for all needed parts rather

Reply:
done :+1: